gqcp/include/Mathematical/Representation/Matrix.hpp

```diff
@@ -169,6 +169,64 @@ class Matrix:
     }
 
 
+    /**
+     *  @param eigenvalues1     the first set of eigenvalues
+     *  @param eigenvalues2     the second set of eigenvalues
+     *  @param tolerance        the tolerance for comparison
+     *
+     *  @return if two sets of eigenvalues are equal within a given tolerance
+     */
+    static bool areEqualEigenvalues(const Matrix<Scalar, Dynamic, 1>& eigenvalues1, const Matrix<Scalar, Dynamic, 1>& eigenvalues2, double tolerance = 1.0e-12) {
+        return eigenvalues1.isApprox(eigenvalues2, tolerance);
+    }
+
+
+    /**
+    *  @param eigenvector1     the first eigenvector
+    *  @param eigenvector2     the second eigenvector
+    *  @param tolerance        the tolerance for comparison
+    *
+    *  @return if two eigenvectors are equal within a given tolerance
+    */
```

Comment:
Maybe as a small guideline for better generated documentation:

```suggestion
    /**
     *  Check if two eigenvectors are equal, within a given tolerance.
     *
     *  @param eigenvector1     The first eigenvector.
     *  @param eigenvector2     The second eigenvector.
     *  @param tolerance        The tolerance for comparison.
     *
     *  @return If two eigenvectors are equal.
     */
```